packages/create-llama/helpers/poetry.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { execSync } from "child_process";
+import fs from "fs";
+
+export function isPoetryAvailable(): boolean {
+  try {
+    execSync("poetry --version", { stdio: "ignore" });
+    return true;
+  } catch (_) {}
```

Comment:
what error handling is used in `callPackageManager`?

Reply:
sorry, i don't get it. Do you mean that we should handle the error explicitly for this function? about callPackageManager we do not catch the error from/in that

Reply:
I mean take a look at `callPackageManager` how they are calling a child process
e.g. they are using `stdio: "inherit"` and checking for the return code which I am not sure is handled here correctly